### ISSUE TITLE
Combo Box selection read when focus enters subsequent edit field

### ISSUE THREAD

**Issue by derekbove** [FreedomScientific/VFO-standards-support#520]
Our team has run into an issue that appears to potentially be a JAWS & Chrome bug. I was unable to reproduce the issue with NVDA, or in FireFox. 

Essentially, when a selection is made in a combo box, closed via the Escape key, then the user navigates via the virtual cursor to the next form field, the combo box contents are read. 

This appears to occur if you close the combo box via the Escape key, then navigate using the virtual cursor. If the Tab key is pressed to move to the next control, the combo box contents are _not_ read. 

To Reproduce:
1. Go to https://indeed.force.com/employerSupport1/s/contactsupport?language=en_US
2. Navigate to the "How Can We Help" combo box using the down arrow
3. On the combo box, press the Enter key to activate the control
4. Use the arrow keys to open the combo box, then choose an item, and press Enter
5. Press Escape to close the combo box (I think this is the root cause of the issue here)
6. Resume virtual keyboard navigation and use the down arrow keys to the First Name label, and then the edit field

### Expected result
The user should be informed that they are in the first name edit. 

### Actual result
JAWS will speak (when entering the First Name edit): "Billing, Edit combo, required, invalid entry"

### Example
See attached video.

https://user-images.githubusercontent.com/83019612/115727175-5f363a80-a351-11eb-85a8-f40310a75213.mp4

### Additional Information
With JAWS 2020, the previous edit field contents are read as you navigate as well. Could not reproduce in JAWS 2021. 

#### JAWS version and build number
2021.2102.34
 
#### Operating System and version
Windows 10 1607

#### Browser and version:
Chrome 89.0.4389.128

**Comment by stevefaulkner** [comment]
@derekbove seems like an edge case to me as the user is in forms mode, why would they press escape in order to move to the next input, as they can do it without pressing ESC?

**Comment by derekbove** [comment]
I agree that this is corner case and would require the user to close the combo box with the escape key, but the fact that it doesn't happen in FireFox, or with NVDA leads me to believe it is either a JAWS bug, or a compatibility issue with JAWS and Chrome in this particular use case with closing a combo box.
